web/core/components/issues/issue-layouts/list/block-root.tsx

```diff
@@ -114,7 +116,7 @@ export const IssueBlockRoot: FC<Props> = observer((props) => {
     issueBlockRef?.current?.classList?.remove(HIGHLIGHT_CLASS);
   });
 
-  if (!issueId) return null;
+  if (!issueId || !issuesMap[issueId].created_at) return null;
```

Comment:
The issueId might not exist in the issueMap, please optionally chain whenever in doubt that it might be undefined to avoid errors